js/components/Toolbar.jsx

```diff
@@ -87,6 +64,7 @@ var Toolbar = React.createClass({
           <Link to="profile" params={{userId: this.state.currentUser._id}}>{this.state.currentUser.firstname}</Link>
```

Comment:
So on the render, when we get state from stores, why don't we have to check if we have this.state.currentUser, this.state.user, etc? Won't render be called before the ajax calls come back, in which case all of the state variables will be undefined?